ui/pages/Search/constants.js

```diff
@@ -5,7 +5,7 @@ import {
 
 export const getSelectedAnalysisGroups = (analysisGroupsByGuid, familyGuids) => Object.values(
   analysisGroupsByGuid,
-).filter(group => group.familyGuids.every(familyGuid => familyGuids.includes(familyGuid)))
+).filter(group => group.familyGuids?.every(familyGuid => familyGuids.includes(familyGuid)))
```

Comment:
nitpick, but this function `getSelectedAnalysisGroups` is used only once now in `PageHeader.jsx`, can you move it there?